src/BundlerMinifierVsix/Commands/ConvertToGulp.cs

```diff
@@ -212,11 +212,14 @@ private static void ModifyPathVariable(ProcessStartInfo start)
 
         private static void StripCommentsFromJsonFile(string fileName)
         {
-            IEnumerable<string> filteredFileContent = from line in File.ReadAllLines(fileName)
-                                                      where !line.TrimStart().StartsWith("//")
-                                                      select line;
+            string fileContent = File.ReadAllText(fileName);
+            var loadSettings = new JsonLoadSettings
+            {
+                CommentHandling = CommentHandling.Ignore
+            };
+            var filteredFileContent = JArray.Parse(fileContent, loadSettings);
 
-            File.WriteAllLines(fileName, filteredFileContent, Encoding.UTF8);
+            File.WriteAllText(fileName, filteredFileContent.ToString(), Encoding.UTF8);
```

Comment:
Let's only write the new file to disk if it differs from `fileContent`. Also, we need to check the file out of SCC before writing the file. The code is `ProjectHelpers.CheckFileOutOfSourceControl(filename)` or something like that